fre/tests/test_files/varlist_local_target_vars_differ

```diff
@@ -0,0 +1,3 @@
+{
+    "sosV2": "sos"
```

Comment:
glad i figured out how to make this work. there was some serious confusion before in the code regarding the way the key and value get used respectively